include/jwt.h

```diff
@@ -25,6 +25,31 @@
 
 #include <stdio.h>
 
+#ifdef _MSC_VER
+
```

Comment:
Please, to match the rest of the coding style, un-indent these defines and remove the extra whitespace just after the #ifdef and before the closing #endif (just looking for consistency here).

Reply:
I'm worrying about that if I remove indentation of this entire code fragment it will become unreadable. Or you want to remove only the first indentation level? How about something like this?

```c++
#ifdef _MSC_VER

#define DEPRECATED(func) __declspec(deprecated) func

#define alloca _alloca
#define strcasecmp _stricmp
#define strdup _strdup

#ifdef JWT_DLL_CONFIG
	#ifdef JWT_BUILD_SHARED_LIBRARY
		#define JWT_EXPORT __declspec(dllexport)
	#else
		#define JWT_EXPORT __declspec(dllimport)
	#endif
#else
	#define JWT_EXPORT
#endif

#else

#define DEPRECATED(func) func __attribute__ ((deprecated))
#define JWT_EXPORT

#endif
```

Reply:
This looks better. I prefer everything left justified, but I'll compromise a wee bit. Alternatively, you could un-nest these. Handle `DEPRECATED` in one block and `JWT_EXPORT` in another.